[2023-02-02T03:33:26.106+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-23T00:00:00+00:00 [queued]>
[2023-02-02T03:33:26.114+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-23T00:00:00+00:00 [queued]>
[2023-02-02T03:33:26.114+0530] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T03:33:26.114+0530] {taskinstance.py:1284} INFO - Starting attempt 3 of 4
[2023-02-02T03:33:26.114+0530] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T03:33:26.131+0530] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract> on 2023-01-23 00:00:00+00:00
[2023-02-02T03:33:26.136+0530] {standard_task_runner.py:55} INFO - Started process 4846 to run task
[2023-02-02T03:33:26.140+0530] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'NSE_Open_Interest', 'extract', 'scheduled__2023-01-23T00:00:00+00:00', '--job-id', '3489', '--raw', '--subdir', 'DAGS_FOLDER/NSE_Open_Interest.py', '--cfg-path', '/tmp/tmpwu489wgd']
[2023-02-02T03:33:26.142+0530] {standard_task_runner.py:83} INFO - Job 3489: Subtask extract
[2023-02-02T03:33:26.196+0530] {task_command.py:389} INFO - Running <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-23T00:00:00+00:00 [running]> on host DESKTOP-SQIOQPD.
[2023-02-02T03:33:26.270+0530] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=nikheleshbhattacharyya@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=NSE_Open_Interest
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2023-01-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-23T00:00:00+00:00
[2023-02-02T03:33:26.271+0530] {logging_mixin.py:137} INFO - 2020-02-14 00:00:00
[2023-02-02T03:35:37.183+0530] {logging_mixin.py:137} INFO - 2020-02-15 00:00:00
[2023-02-02T03:35:40.294+0530] {logging_mixin.py:137} INFO - 2020-02-16 00:00:00
[2023-02-02T03:35:43.376+0530] {logging_mixin.py:137} INFO - 2020-02-17 00:00:00
[2023-02-02T03:35:43.683+0530] {logging_mixin.py:137} INFO - 2020-02-18 00:00:00
[2023-02-02T03:37:55.426+0530] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/root/airflow/dags/NSE_Open_Interest.py", line 83, in extract
    dd,values = get_data(date)
TypeError: cannot unpack non-iterable int object
[2023-02-02T03:37:55.431+0530] {taskinstance.py:1322} INFO - Marking task as UP_FOR_RETRY. dag_id=NSE_Open_Interest, task_id=extract, execution_date=20230123T000000, start_date=20230201T220326, end_date=20230201T220755
[2023-02-02T03:37:55.445+0530] {standard_task_runner.py:100} ERROR - Failed to execute job 3489 for task extract (cannot unpack non-iterable int object; 4846)
[2023-02-02T03:37:55.468+0530] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-02-02T03:37:55.487+0530] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
